### DISCUSSION TITLE
Rockchip ARM SoC with AIO ?

### DISCUSSION THREAD

**Discussion by docjyJ** [nextcloud/all-in-one#4837] [Questions] [answered]
Hey,

I would like to host my Nextcloud server directly at home.
I looked at ARM cards, and found the Rockchip RK3588 which looks good.

They are compatible with Debian, like Raspberry (and they seem to perform better than Raspberry).

I think the AIO runs on it but what do you think?

Thank,

See : [Raspberry Pi 5 contre Orange Pi 5 Plus contre Rock 5 Modèle B](https://picockpit.com/raspberry-pi/raspberry-pi-5-vs-orange-pi-5-plus-vs-rock-5-model-b/)

**Comment by szaimen** [comment]
Hi, this should work fine as long as the board has an arm64 processor. See https://github.com/nextcloud/all-in-one?tab=readme-ov-file#which-cpu-architectures-are-supported

**Comment by BraidenPsiuk** [comment, marked as answer]
I can't speak to the 3588, but if you want a dedicated ARM board for primarily running AIO and not much else, I can personally recommend the [ROCKPro64](https://pine64.org/devices/rockpro64/) by Pine64 for running it at home. But only the 4GB model, I wouldn't bother with the 2GB one. They have a nice [metal NAS case](https://pine64.com/product/rockpro64-metal-desktop-nas-casing/) if you plan to add extra storage to it. Their boards generally run Linux really well, and like- clean distributions from official or community sources, not shady distributions downloaded from some random Google Drive link like the Orange/Banana Pis offer :) 4GB is still not quite enough if you want to play around with the AI models and add a ton of extra containers- but if you're just using it as a storage server with a few extras enabled like Nextcloud Office, it should fit those needs perfectly. Edit: here's [all the supported distros](https://pine64.org/documentation/ROCKPro64/Software/Releases/), Debian is included

**Reply by docjyJ** [reply]
I would like to run Nextcloud Office, Nextcloud Assistant and a mail server (Salwart). This is why I am considering the RK3588.

The price is still lower than the X86 in my opinion, and the consumption is advantageous.

Thank you very much, if you have good feedback with the RK3399, it should be just as good with the RK3588!
